.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+# based on: https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/.circleci/config.yml
```

Comment:
I see that the [original also uses `tflint`](https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/.circleci/config.yml#L25) which has additional tests (e.g. invalid EC2 instance type).

I wonder if it may be useful for as well.